lib/widgets/profile.dart

```diff
@@ -0,0 +1,259 @@
+import 'dart:convert';
+
+import 'package:flutter/material.dart';
+
+import '../api/model/model.dart';
+import '../model/content.dart';
+import '../model/narrow.dart';
+import 'content.dart';
+import 'message_list.dart';
+import 'page.dart';
+import 'store.dart';
+
+class _TextStyles {
```

Comment:
I've seen in other places usage of private `_kFooBar` variables used for reusable style following their 'Begin global constant names with prefix "k"' rule, but the same rule recommends using a container class over that. I do personally prefer this style, so I elected for refactor in this way for now.

Reply:
Sure, this approach looks good to me.